src/qml/controls/ValueInput.qml

```diff
@@ -36,6 +36,14 @@ TextInput {
                 enabled: false
                 textColor: Theme.color.neutral4
             }
+        },
+        State {
```

Comment:
Remove this state as its not appropriate

Reply:
done